lms/djangoapps/courseware/date_summary.py

```diff
@@ -240,6 +256,11 @@ def is_enabled(self):
 
     @lazy
     def date(self):
+        if self.course_overview.self_paced and self.enrollment:
```

Comment:
We should probably put some kind of kill switch in here. Maybe check to see if the config model is enabled?